app/__init__.py

```diff
@@ -60,14 +60,16 @@ def create_app(debug=False):
     )
 
     # create subdirectories if they don't already exist
-    create_dir(app.config['RECIPES_PATH'].joinpath('pico'))
-    create_dir(app.config['RECIPES_PATH'].joinpath('zymatic'))
+    create_dir(app.config['RECIPES_PATH'].joinpath('pico/archive'))
+    create_dir(app.config['RECIPES_PATH'].joinpath('zymatic/archive'))

```

Comment:
Is the `zseries` and `zseries/archive` paths not also dynamically created if not already created?

Reply:
thanks for the catch, updating now to include